sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -96,11 +99,12 @@ def write_record(cls, file_handle, value):
       value: A string content of the record.
     """
     encoded_length = struct.pack('<Q', len(value))
```

Comment:
I think we need to take a closer look at this IO regarding `str` vs `bytes`:
- we may want to compute encoded_length after `value` is encoded to bytes.
- we need to clarify in the docstring what type is returned as a result of `read_record` and what type is expected as argument to `encoded_num_bytes`. 
- we should also have unicode strings in tfrecordio_test.